src/rez/rezconfig.py

```diff
@@ -914,6 +914,8 @@
     },
 ]
 
+optionvars = None
```

Comment:
There needs to be a docstring here, see the comments on all other settings. These get parsed and turned into the `Configuring Rez` wiki page.